Nodejs/Product/Npm/SPI/NpmInstallCommand.cs

```diff
@@ -2,15 +2,20 @@
 
 namespace Microsoft.NodejsTools.Npm.SPI
 {
-    internal class NpmInstallCommand : NpmCommand
+    public class NpmInstallCommand : NpmCommand
     {
+        public static bool IsInstalling { get; private set; } = false;
```

Comment:
>IsInstalling [](start = 27, length = 12)

What happens if the debugging session checks the flag and *then* npm starts installing?

Reply:
This scenario cannot happen because starting npm install happens before loading the project. Check comment on `NpmWizardExtension.cs:27`

```
// Create the "node_modules/@types" folder before opening any files (which creates the
// context for the project). This allows tsserver to start watching for type definitions
// before any are installed (which is needed as "npm install" runs async, so the modules
// usually aren't installed before the project context is loaded).
```